mgmt/imgmgr/syscfg.yml

```diff
@@ -50,6 +50,9 @@ syscfg.defs:
     IMGMGR_LOG_LVL:
         description: 'Minimum level for the image manager structured log.'
         value: 15  # Log disabled by default.
+    IMGMGR_DUMMY_HDR:
+        description: 'Returns dummy image header data for imgr functions'
```

Comment:
I think this setting needs a bit more explanation.  Maybe add "Useful when there are no images present, e.g., unit tests."